README.md

```diff
@@ -14,23 +30,73 @@ A lightweight extension to run and debug Java test cases in Visual Studio Code.
 
 The [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) works with [Language Support for Java by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java) and [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) to provide the following features:
 
-- Run test cases
-- Debug test cases
+- Run/Debug test cases
+- Customize test configurations
 - View test report
 - View tests in Test Explorer
+- Show test logs
 
-## Usage
+
+## Requirements
+
+- JDK (version 1.8.0 or later)
+- VS Code (version 1.23.0 or later)
+- [Language Support for Java by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java)
+- [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug)
+
+## Quickstart
 
 ![Run/debug JUnit test](demo/demo.gif)
 
-## Available commands
+## Features
+
+### Run/Debug through Code Lens
```

Comment:
The feature should be "Run/Debug Test Cases". Code Lens can be mentioned as a way of implementation, like what's in the context menu.

Reply:
Addressed.